opba-db-migration/src/main/java/de/adorsys/opba/config/MigrationConfig.java

```diff
@@ -0,0 +1,9 @@
+package de.adorsys.opba.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackageClasses = de.adorsys.opba.config.hibernate.PrefixAndSnakeCasePhysicalNamingStrategy.class)
```

Comment:
This means to scan 
```
de.adorsys.opba.config.hibernate
```
Maybe for migration we can have more explicit package?
like 
de.adorsys.opba.config.migration
and
de.adorsys.opba.config.migration.hibernate

Reply:
done